CashBudTests/TransactionMapperTest.swift

```diff
@@ -0,0 +1,33 @@
+import XCTest
+@testable import CashBud
+
+class TransactionMapperTest: XCTestCase {
+    var transactionMapper: TransactionMapper!
+    
+    override func setUp() {
+        super.setUp()
+        transactionMapper = TransactionMapper()
+    }
+    
+    override func tearDown() {
+        transactionMapper = nil
+        super.tearDown()
+    }
+    
+    func testMapTransaction() {
+        // Test case for mapping a transaction
+        let transaction = Transaction(id: "1", amount: 10.0, date: Date())
+        let mappedTransaction = transactionMapper.mapTransaction(transaction)
```

Comment:
These two lines throw errors: "Extra arguments at positions #1, #2, #3 in call", "Missing argument for parameter 'from' in call", "Value of type 'TransactionMapper' has no member 'mapTransaction'"